dags/socorro_import.py

```diff
@@ -18,39 +39,155 @@
 
 dag = DAG("socorro_import", default_args=default_args, schedule_interval="@daily")
 
-# input: crashstats-telemetry-crashes-prod-us-west-2/v1/crash_report
-# output: telemetry-parquet/socorro_crash/v2
-crash_report_parquet = EMRSparkOperator(
+# Unsalted cluster name so subsequent runs fail if the cluster name exists
+cluster_name = 'socorro-import-dataproc-cluster'
+
+# Defined in Airflow's UI -> Admin -> Connections
+gcp_conn_id = 'google_cloud_airflow_dataproc'
+connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
+
+# Required to copy socorro data from s3 to gcs
+read_aws_conn_id='aws_socorro_readonly_s3'
+
+# Required to write parquet data back to s3://telemetry-parquet
+write_aws_conn_id='aws_dev_socorro_telemetry_parquet_s3'
+
+gcstj_object_conditions = {
+    'includePrefixes':  'v1/crash_report/{{ ds_nodash }}'
+}
+
+gcstj_transfer_options = {
+    'deleteObjectsUniqueInSink': True
+}
```

Comment:
If the variables are not used more than once, I think it makes sense to put them next to the operator they take place in. In particular, `gcstj_object_conditions` and `gcstj_transfer_options` don't have any real meaning outside of the gcs transfer operator.